script/ghci-flags

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+# Computes the flags for ghcide to pass to ghci. You probably won’t be running this yourself, but rather ghcide will via configuration in hie.yaml.
+
+set -e
+
+cd $(dirname "$0")/..
+
+root="$(pwd)"
+ghc_version="$(ghc --numeric-version)"
+
+if [ "$1" == "--builddir" ]; then
+  repl_builddir="$2"
+  shift 2
+else
+  repl_builddir=dist-newstyle
+fi
```

Comment:
We can’t pass flags into this program from `hie.yaml`, but I wanted to reuse it from `script/repl` without duplicating much of it or adding redundant wrapper scripts, so I’ve made this default to `dist-newstyle` if it’s unspecified.